app/src/main/java/com/infomaniak/swisstransfer/ui/components/IllustratedMessageBlock.kt

```diff
@@ -42,34 +42,40 @@ import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
 
 @Composable
 fun IllustratedMessageBlock(
-    icon: ImageVector,
-    @StringRes title: Int,
-    description: String,
+    icon: ImageVector?,
+    @StringRes title: Int?,
+    description: String?,
     modifier: Modifier = Modifier,
 ) {
     Column(
         modifier = modifier,
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
-        Image(imageVector = icon, contentDescription = null)
-
-        Spacer(Modifier.height(Margin.Huge))
+        icon?.let {
+            Image(imageVector = it, contentDescription = null)
```

Comment:
We lost the Image in ReceivedScreen on mobile.
<img width="528" alt="image" src="https://github.com/user-attachments/assets/c3ab9de7-78a7-4cd0-8faf-791ef23f7f81">

Reply:
I've added the image on mobile.